guide/additional-info/changes-in-v12.md

```diff
@@ -1,12 +1,12 @@
 # Updating from v11 to v12
 
-After a long time in development, Discord.js v12 is nearing a stable release, meaning it's time to update from v11 to get new features for your bots!  However, with those new features comes a lot of changes to the library that will break code written for v11.  This guide will serve as a handy reference for updating your code, covering the most commonly-used methods that have been changed, new topics such as partials and internal sharding, and will also include a comprehensive list of the method and property changes at the end.
+After a long time in development, Discord.js v12 has been a stable release, meaning it's time to update from v11 to get new features for your bots!  However, with those new features comes a lot of changes to the library that will break code written for v11.  This guide will serve as a handy reference for updating your code, covering the most commonly-used methods that have been changed, new topics such as partials and internal sharding, and will also include a comprehensive list of the method and property changes at the end.
```

Comment:
Slightly weird wording in my opinion, should be something along the lines of
`After a long time in development, Discord.js v12 has been released and is now stable...`

Reply:
thanks!